backend/routes/user_routes.py

```diff
@@ -43,18 +40,14 @@ async def send_login_link(
     user.login_code = str(login_code_uuid)
     session.commit()
 
+    if environ.get("TESTING"):
```

Comment:
I have a preference to not have test code in the system, but if you do decide to keep it, the functionality and environment variable should be documented in the readme and in `env.template`.

Reply:
This automatically added by `conftest` when we run `test.sh`. So we doesn't need to add it into `env.template`.

Reply:
Right, makes sense. Then it should still be in `readme.md`. :)

Reply:
Why that need to be in `readme.md`?

Reply:
So that this is clear to others looking at the code. This is a public repo, so we have to guide people who are not in the project too.

Reply:
Hi @wayangalihpratama, you can explain in the README that we added 'TESTING' to conftest.py to mock or skip certain steps related to a third-party service. Also, please note that 'TESTING' should not be included in the Docker Compose environment.

cc @kjkoster